_gsocproposals/proposal_TRANSFER.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Next generation of CMS data replication system
+layout: plain
+project: TRANSFER
+organization: Cornell, USA and Univ. of Bologna, Italy
```

Comment:
can you change that to CMS? And I will add CMS to the participating organizations on http://hepsoftwarefoundation.org/activities/gsoc.html